diff --git a/node_modules/postcss-px-to-viewport/index.js b/node_modules/postcss-px-to-viewport/index.js
index 5556893..47dde28 100644
--- a/node_modules/postcss-px-to-viewport/index.js
+++ b/node_modules/postcss-px-to-viewport/index.js
@@ -97,7 +97,7 @@ module.exports = postcss.plugin('postcss-px-to-viewport', function (options) {
     });
     
     if (landscapeRules.length > 0) {
-      var landscapeRoot = new postcss.atRule({ params: '(orientation: landscape)', name: 'media' });
+      var landscapeRoot = new postcss.AtRule({ params: '(orientation: landscape)', name: 'media' });
       
       landscapeRules.forEach(function(rule) {
         landscapeRoot.append(rule);
